src/front-end/src/components/MentorHome.js

```diff
@@ -47,14 +47,19 @@ class MentorHome extends Component {
         >
           Accepted Mentees
         </Button>
+        {/* replace what's in square brackets [] w/ info from DB */}
+        <Button
+          type="primary"
+          htmlType="button"
+          id="logout_button"
+          // href="[insert logout page path here]"
+        >
+          logout
+        </Button>
```

Comment:
Can you add the onClick event property to the button? Otherwise the log-out function won't work (i.e onClick={this.onLogout})

Reply:
Added!